src/GraphQL.Tests/Types/AllScalarGraphTypeTests.cs

```diff
@@ -19,20 +19,18 @@ public class AllScalarGraphTypeTests
          *    UIntGraphType
          *    LongGraphType
          *    ULongGraphType
-         *    
+         *
          *    BooleanGraphType
          *    FloatGraphType
          *    StringGraphType
-         *    
-         *  Does not test:
```

Comment:
Why remove this comment? Does it test all the semantics of the mentioned types? For example, that the proper exception is returned for deserializing an invalid date type? Or overflow conditions for a decimal? I don’t think so. Initially it didn’t test these hardly at all because these types cannot be inlined as an attribute.  (Not at a pc just now but going from memory. )

Reply:
Was this comment for single test or a whole file?

Reply:
For the four types listed right under it.

Reply:
For the whole file

Reply:
ok